.changeset/sad-taxis-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+`wrangler deploy` is able to deploy new container versions
```

Comment:
are containers a beta/private feature?

Reply:
Yup, private now, public in June. https://blog.cloudflare.com/cloudflare-containers-coming-2025/